src/index.ts

```diff
@@ -7,3 +10,12 @@ app.listen(env.server.port, () => {
   console.log(apiClients.sendgridApiClient);
   console.log(`🚀 App listening on port ${env.server.port}!`);
 });
+
+(async function() {
+  const ngrokUrl = await ngrok.connect();
+  console.log(
+    `[Instructions to test webhook]
+    Go to https://larivercorp.nationbuilder.com/admin/webhooks/new and
+    add a new webhook with Endpoint URL: ${ngrokUrl}/webhooks/nationbuilder/personCreated.`,
+  );
+})();
```

Comment:
This should only happen when `process.env.NODE_ENV === 'development'`. Would you mind adding a `nodeEnv` to `/src/env.ts` and adding it here?